core/src/main/resources/jenkins/model/Jenkins/legend_es.properties

```diff
@@ -20,17 +20,17 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-grey=El proyecto no se ha ejecutado nunca o est� deshabilitado.
-grey_anime=Se est� ejecutando el proyecto por primera vez.
+grey=El proyecto no se ha ejecutado nunca o est\u00E1 deshabilitado.
+grey_anime=Se est\u00E1 ejecutando el proyecto por primera vez.
 blue=La \u00FAltima ejecuci\u00F3n fue correcta.
 blue_anime=La \u00FAltima ejecuci\u00F3n fue correcta. Se ha lanzado una nueva ejecuci\u00F3n.
 yellow=La \u00FAltima ejecuci\u00F3n fue correcta pero inestable. Esto es debido normalmente a fallos en algunos tests.
 yellow_anime=La \u00FAltima ejecuci\u00F3n fue correcta pero inestable. Se ha lanzado una nueva ejecuci\u00F3n.
-red=La �ltima ejecuci�n fall�.
-red_anime=La �ltima ejecuci�n fall�. Se ha lanzado una nueva ejecuci�n.
+red=La \u00FAltima ejecuci\u00F3n fall\u00F3.
+red_anime=La \u00FAltima ejecuci\u00F3n fall\u00F3. Se ha lanzado una nueva ejecuci\u00F3n.
 
-health-81plus=El estado de salud del proyecto supera el 80%. Puedes situar el rat�n sobre el icono para ver una explicaci�n detallada.
-health-00to20=El estado de salud del proyecto es inferior al 20%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-41to60=El estado de salud del proyecto est\u00E1 entre el 40% y el 60%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-21to40=El estado de salud del proyecto est\u00E1 entre el 20% y el 40%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-61to80=El estado de salud del proyecto est\u00E1 entre el 80% y 80%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
+health-81plus=El estado de salud del proyecto supera el 80%. Puedes situar el ratón sobre el \u00EDcono para ver una explicaci\u00F3n detallada.
```

Comment:
Oh, you are right http://dle.rae.es/?id=KsRzX3u `ícono` is accepted too (only in America). I've never seen it accented BTW (even in LATAM writings).